rust/tools/authorization-logic/src/parsing/AuthLogic.g4

```diff
@@ -120,10 +139,19 @@ ATTRIBUTE: 'attribute';
 
 // Identifiers wrapped in quotes are constants whereas
 // identifiers without quotes are variables.
-ID : ('"')? [_a-zA-Z0-9/.#:]* ('"')?;
+ID : ('"')? [_a-zA-Z]+ [_a-zA-Z0-9/.#:]* ('"')?;
+NUMLITERAL : [0-9]+;
```

Comment:
Do you need negatives? Or is that handled elsewhere?

Reply:
It's not needed for the transparent release demo certainly. I don't recall seeing that souffle supports negative numeric literals. If souffle does not support this, I don't want to bother coming up with some way to support them now given that we don't need them yet. (Supporting negatives could be useful if we keep some extension of this grammar around maybe)